src/components/card/asCardChild.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react';
+import CardContext from './CardContext';
+
+function asCardChild(WrappedComponent) {
+  class CardChild extends Component {
```

Comment:
consider simplify it by replacing class with a function

Reply:
Done